source/tryGoal/component/shared/questionContext.html

```diff
@@ -0,0 +1,31 @@
+<div class="question-context-inner-wrapper" ng-show="display === 'secondForm' || display === 'firstForm' || display === 'error'">
+    <div class="question-context-response-container" style="margin: 10px;">
+
+        <!--  Technical error -->
+        <div ng-if="display === 'error'">
+            <p>Sorry, but we are unable to show additional information for this question.</p>
```

Comment:
Hi Ellie, is this the area where we plan to show detailed error information in the future?  If so, this new panel wouldn't always contain question context information but be a more general 'query context' panel.  What do you think?

Reply:
Hey Will, I'm not too sure on what we intend this to be in the future but it seems like a good place to provide more detailed error information later on. Shall I change the wording to 'Query Context' to open the scope of it a bit more?